src/components/vectorfeedback/partials/vectorfeedback.html

```diff
@@ -1,39 +1,39 @@
 <div class="ga-vector-feedback-panel collapse in" aria-expanded="true">
     <a class="ga-vector-feedback-header" data-toggle="collapse" data-target="#ga-feedback-vector-body" aria-expanded="true">
-        <div class="ga-vector-feedback-title ga-no-select pull-left">Vector Tiles Alpha Viewer</div>
+        <div class="ga-vector-feedback-title ga-no-select pull-left">Vector Tiles BASEMAP & HYBRID</div>
         <i class="fa fa-caret-down"></i>
     </a>
     <div id="ga-feedback-vector-body" class="collapse in">
         <p>
-            Lorem ipsum dolor sit amet, nam an veritus adipiscing, mutat habemus sea an. Ne decore nemore scaevola mel,
-            vim at qualisque consulatu. Pro ponderum molestiae dignissim et. Est id ferri meliore atomorum, in mei
-            primindoctum. Ea sint dolore nec, ea dicam propriae verterem est.
+            The map consists of so-called <b>vector tiles</b> instead of raster tiles. The map appears faster on the screen and you can easily adjust the style. The map is drawn directly in your browser.
+            <br>
+            The data is retrieved once, but you can apply an infinite number of styles.To make the styling and loading of the map even easier and faster, the data is organized in a simpler way. With just a few themes you can now quickly and easily create your own map of the whole of the Netherlands at every zoom level! Just try it out:
```

Comment:
ok, you re lucky I was gona push force. Try to create a PR if possible, this can be easily lost otherwise if I rebase and push force. and when I pull it gets messy. thanks though

Reply:
@davidoesch :laughing: whole of the Netherlands ??

Reply:
https://github.com/geoadmin/mf-geoadmin3/blob/e8b68e0f9a8334b0bc8392e29fbda6a50dfd654c/src/components/vectorfeedback/partials/vectorfeedback.html next time i will directyl push to master